### PR TITLE
Closes AgileVentures/MetPlus_tracker#623

### PR DESCRIPTION
**Pull request by hin101**
Functionality done but having issues with Cucumber Tests.

### CODE REVIEW

File: app/assets/javascripts/jobs.js [hunk 1]
```diff
@@ -131,3 +131,7 @@ $(function () {
   });
 
 });
+
+$(function() {
+  $(".job-type-multiple").select2();
```
Comment (patmbolger) [inline app/assets/javascripts/jobs.js]: Don't need a separate DOM-ready function for this.  Could add this statement to the existing function - see similar statements starting at line 93.

File: app/models/job.rb [hunk 2]
```diff
@@ -4,11 +4,11 @@ class Job < ActiveRecord::Base
   belongs_to :company_person
   belongs_to :address
   belongs_to :job_category
+  has_and_belongs_to_many :job_types
 
   has_many   :job_skills, inverse_of: :job, dependent: :destroy
   has_many   :skills, through: :job_skills
-  accepts_nested_attributes_for :job_skills, allow_destroy: true,
-                                             reject_if: :all_blank
+  accepts_nested_attributes_for :job_skills, allow_destroy: true
```
Comment (patmbolger) [inline app/models/job.rb]: why drop the `reject_if:` argument?

File: db/migrate/20170814193715_add_default_job_types.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class AddDefaultJobTypes < ActiveRecord::Migration
```
Comment (patmbolger) [inline db/migrate/20170814193715_add_default_job_types.rb]: Unless we need to convert existing data in the DB, I would rather _not_ add data to a table with a migration (which should be limited to schema changes if at all possible).

I suggest adding this data create logic to `seeds.rb` (the part where we seed "production" data (create default Agency, create agency and company roles).

File: app/models/job.rb (hunk 2 above)
Comment (hin101) [inline app/models/job.rb]: Ah ill add it back in I think I accidently deleted this line

**Comment by patmbolger** [comment]
Looks good, and thanks for making the requested changes.

**Pull request merged**